README.md

```diff
@@ -327,7 +327,8 @@ Statics parameters
   - Example: `library-name/example.js:lib/example.min.js` (customizes both file path and file name of minified file)
 - `jsCompiledOutput`: Where to place compiled JS files. This folder will be made public by default.
   - Default: `.build/js`
-
+- `nodeEnv`: Sets environment variable in development or production mode. Default is handled by the script
```

Comment:
change this to `nodeEnv`: Param to override the `NODE_ENV` environment variable.